BraveShared/Preferences.swift

```diff
@@ -26,8 +26,8 @@ public class Preferences {
 }
 
 extension Preferences {
-    final class DAU {
-        static let lastLaunchInfo = Option<[Int?]?>(key: "dau.last-launch-info", default: nil)
+    public final class DAU {
```

Comment:
The preference we need to check here is in a framework so I had to add some `public` keywords to make it acceesible.
I don't think we should have preferences in a framework tbh.

Reply:
Yeah we would need to re-structure some of the stuff in `BraveShared` (`DAU` and `URP` for example) to either not live there or open them up to accept these preferences from `Client` side. Obviously out of scope for this ticket though :)

Reply:
Oh fine.